e2etest/zt_managed_disks_test.go

```diff
@@ -147,3 +147,61 @@ func TestManagedDisks_OAuthRequired(t *testing.T) {
 		"",
 	)
 }
+
+// Test a managed disk of size 7.9 TB
+func TestManagedDisks_LargeManagedDisk(t *testing.T) {
```

Comment:
Instead of adding to the legacy e2e tests I'd like to start seeing people use the new e2e test framework.